src/platform/packages/shared/kbn-alerts-ui-shared/src/common/apis/fetch_alerts_deletion_settings/update_alerts_deletion_settings.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { HttpSetup } from '@kbn/core/public';
+import { AsApiContract, RewriteRequestCase } from '@kbn/actions-plugin/common';
+import type {
+  RulesSettingsAlertsDeletion,
+  RulesSettingsAlertsDeletionProperties,
+} from '@kbn/alerting-types/rule_settings';
+// import { INTERNAL_BASE_ALERTING_API_PATH } from '../../constants';
```

Comment:
same here. This file is just working as a mock until we do the actual settings update